src/catalog/src/system_schema/information_schema/flows.rs

```diff
@@ -290,6 +309,42 @@ impl InformationSchemaFlowsBuilder {
                     input: format!("{:?}", flow_info.options()),
                 },
             )?));
+        self.created_time
+            .push(Some(flow_info.created_time().timestamp_millis().into()));
+        self.updated_time
+            .push(Some(flow_info.updated_time().timestamp_millis().into()));
+        self.last_execution_time.push(
+            flow_info
+                .last_execution_time()
+                .map(|t| t.timestamp_millis().into()),
+        );
+
+        let mut source_table_names = vec![];
+        let catalog_name = self.catalog_name.clone();
+        let catalog_manager = self
+            .catalog_manager
+            .upgrade()
+            .context(UpgradeWeakCatalogManagerRefSnafu)?;
+        for schema_name in catalog_manager.schema_names(&catalog_name, None).await? {
+            let mut stream = catalog_manager.tables(&catalog_name, &schema_name, None);
+
+            while let Some(table) = stream.try_next().await? {
```

Comment:
The loop may slow down with numerous tables. We could implement a method in catalog_manager to check for a table's existence by its ID. @WenyXu @fengjiachun

Reply:
Seems we need a method `tables_by_ids`?

Reply:
#5645